app/policies/consumer_role_policy.rb

```diff
@@ -50,6 +50,14 @@ def edit?
     return false
   end
 
+  # Checking presence of hbx_staff_role and if identity_validation is valid. If either are true,
+  # then the user has access to continue past RIDP.
+  def accessible?
+    person = @user.person
+    return true if person && (@user.has_hbx_staff_role? || person.consumer_role.identity_validation == 'valid')
```

Comment:
We should favor methods that are defined on the model as opposed to recreating logic outside of it (esp. with hardcoded string values).  ConsumerRole has a method that would be useful here: `identity_verified?`

Also, I think we can just return the expression:
```
person && (@user.has_hbx_staff_role? || person.consumer_role.identity_validation == 'valid')
```
This is the same as returning `true` if true, otherwise `false`.